ECOSYSTEM.md

```diff
@@ -1,27 +1,36 @@
+We aim to build a vibrant and inclusive ecosystem of partners, core contributors, and community that goes beyond the main React Native GitHub repository. This document explains the roles and responsibilities of various stakeholders and provides guidelines for the community organization. The structure outlined in this document has been in place for a while but had not been written down before.
 
 There are three types of stakeholders:
 
-* **Partners:** Companies that are significantly invested in React Native and have been for years.
+* **Partners:** Companies significantly invested in React Native and take responsibility for the React Native vision and community.
 * **Core Contributors:** Individual people who contribute to the React Native project.
 * **Community Contributors:** Individuals who support projects in the [react-native-community](https://github.com/react-native-community) organization.
 
 ## Partners
 
-Partners are companies that are significantly invested in React Native and have been for years. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Partners think of React Native as a product: they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications.
+Partners are companies that are significantly invested in React Native and demonstrate ownership. Informed by their use of React Native, they push for improvements of the core and/or the ecosystem around it. Examples of this may include large scale contributions to `react-native` or owning essential tools or libraries. 
 
-React Native's current set of partners include Callstack, Expo, Infinite Red, Meta, Microsoft and Software Mansion. Many engineers from these companies are core contributors, and their partner responsibilities also include:
+Partners think of React Native as a product; they understand the trade offs that the project makes as well as future plans and goals. Together we shape the vision for React Native to make it the best way to build applications. 
 
-* **[Callstack](https://callstack.com/):** Manages releases, maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and organizes [React Native EU](https://react-native.eu/)
-* **[Expo](https://expo.dev/):** Builds [expo](https://github.com/expo/expo) and [EAS](https://expo.dev/eas) on top of React Native to simplify app development
+Baseline responsibilities of partners include: 
+* Attending monthly meeting
+* Contributing to the release process. Examples include being a [community releaser](https://reactnative.dev/contributing/release-roles-responsibilities#release-role-2-community-releaser), testing new releases, technical support for release issues.
+* Engage on the core contributor Discord
+
+Our current partners and other areas of ownership:
+* **[Coinbase](TODO ):** TODO
```

Comment:
Is Coinbase going to continue to invest in this? I know at least some of their internal RN champions were laid off earlier this year.

Reply:
Yea that's my question as well. We can discuss this in our next meeting. I think if I don't hear word back, will remove them from this list

Reply:
👋 Hi team! Brent here from Coinbase. We're down, but not out 😄 .

We're ramping back up, and hopefully y'all have been seeing Bruno in the weekly RN release meetings.

Reply:
Also, it's worth mentioning we are getting close to open-source some of our tooling around measuring React Native performance.

For now, we shared a bit of a tease in this recent article https://blog.coinbase.com/performance-vitals-a-unified-scoring-system-to-guide-performance-health-and-prioritization-127f80268ee4, and more to come soon 😁

Reply:
Additional work we are driving:
- Helping out migrating `@react-native-community/datetimepicker` to be Fabric compatible (and possibly others modules in the upcoming months)
- Investigating enabling the new architecture in our RN app, and will be happy to write a blogpost documenting the process and results once the project is completed
- Can help out with

Reply:
Hey @lunaleaps, here our suggestion:

> **[Coinbase](https://coinbase.com/):** Builds world class crypto apps in React Native. Publishes [articles](https://blog.coinbase.com/tagged/react-native) to advertise React Native development. Supports OSS libraries on their migration to the new architecture.